class RetryException(Exception):
    pass


class MaxRetryException(Exception):
    pass
